runtime/vm/classloadersearch.c

```diff
@@ -237,3 +241,71 @@ addZipToLoader(J9JavaVM * vm, const char * filename, J9ClassLoader * classLoader
 
 	return rc;
 }
+
+jclass
+getJimModules(J9VMThread *currentThread) {
```

Comment:
Brace on it's own line,

Reply:
> Brace on it's own line,
No apostrophe in the possessive of "it".